docs/tutorials/working-with-llms.md

```diff
@@ -0,0 +1,227 @@
+
+- Select a model in `GGUF` format on Huggingface and navigate to the model's page.
+- On the page, Click on the `Use this model` button and select `Ollama` in the menu.
+- From the pop-up dialog, take the full model name along with the tag. For example, the full model name shown in the image below is `hf.co/QuantFactory/Ministral-3b-instruct-GGUF:Q4_K_M`.
+
+:::tip
+Pick the `Q4_K_M` quantization of the `GGUF` models on Huggingface, as this variant generally seems to strike a good balance between the model size and the performance for local usage.
+:::
+
+![](../img/working-with-open-webui/gguf-model-page-hf.png)
+
+- Navigate to the Models page in the Open WebUI, `Admin Panel > Settings > Models`.
+- Enter the full model name in the `Pull a model from Ollama.com` field and click on the download button. Once the model is downloaded, you can use it with the Chat and the Playground features.
+
+![](../img/working-with-open-webui/download-gguf-from-hf.png)
+
+### Using OpenAI-compatible APIs
+
+You can use OpenAI-compatible APIs from a local server such as [localai](https://localai.io/), [llamafile](https://github.com/Mozilla-Ocho/llamafile), or cloud providers such as [Groq](https://groq.com/), [OpenAI](https://openai.com/api/), etc.
+
+For example, you can set up the OpenAI-compatible API from [Groq](https://groq.com/) by following the steps below:
+
+- Create a Groq cloud account at https://console.groq.com.
+- Create an API key at https://console.groq.com/keys.
+- Navigate to `Admin Panel > Settings > Connections` in the Open WebUI extension and add Groq as an OpenAI-compatible API provider. Use `https://api.groq.com/openai/v1` as the base URL and use the API key generated in the previous step.
+
+You should be able to see the available models from Groq in the model selector drop-down.
+
+![](../img/working-with-open-webui/openai-api-setting.png)
+
+![](../img/working-with-open-webui/groq-models-list.png)
+
+### Interacting with the local LLMs
+
+Open WebUI provides a couple of GUI features to let you interact with the local LLMs.
+
+#### Chat Interface
+
+Open WebUI provides a ChatGPT-style chat interface to interact with the local LLMs. 
+
+- Select the LLM you want to interact with in the LLM selector dropdown. 
+
+:::tip
+- Set your preferred LLM model as default to persist the model selection between Rancher Desktop's restarts.
+- You can interact with multiple models at the same time. You can select multiple LLM models using the `+` button next to the model selector dropdown. 
+:::
+
+- You can now chat with the LLM using the chat interface.
+
+![](../img/working-with-open-webui/multi-model-chat.png)
+
+One of the interesting features of the OpenWebUI's chat interface is that when the chat response contains HTML, CSS, and JS code snippets, a live preview (static HTML) of the code is generated, which you can also interactively update by modifying and saving the code in the code snippets.
```

Comment:
I agree. May be  `static web page` might be a better term. By definition, a static web page is a page with fixed content and the fixed content can have JS too. 

In any case, I accept your suggestion to remove `static HTML` to avoid confusion.